### ISSUE TITLE
Would be possible to create NFTs while minting (or just before the mint)? like a collection with infinite NFTs

### ISSUE THREAD

**Issue by SoftwareMechanic** [metaplex-foundation/sugar#481]
### Feature

Hello everyone.
I don't think this is a feature request, and it is not a bug.

it's just me trying to understand something which seems to be not very common. and found lots of dead links while looking for it.
Please bear with me, I am new to Solana and Metaplex.

I understand how to create an NFT predefined collection and to mint from it.

But I am trying to understand how I could create a collection of infinite NFTs, in which the metadata are created/set just before the mint.

### Ideal solution or implementation

basically I need an infinite (or very large) collection which allow the users to define some attributes of the NFT before mint it.

### Other context

is this possible right? if yes, can you please indicate to me where I can have more info about it or samples which do something similar?

Thank you so much in advance

**Comment by SoftwareMechanic** [comment]
Solved!

Basically I just had to create my own candy machine using `Hidden settings` and modify the NFT metadata once minted.

thanks to Mark Sackerberg on Discord for pointing me to the right direction.
